graphql_compiler/tests/test_ir_generation_errors.py

```diff
@@ -620,12 +620,6 @@ def test_filter_directive_wrong_location(self):
                     description @output(out_name: "description_text")
                 }
             }''',
-            '''{
-                Location @filter(op_name: "name_or_alias", value: ["$foo"]) {
-                    name @output(out_name: "name")
-                }
-            }''',
-
```

Comment:
Changed the fields of Location slightly. Anyways, the few lines above cover the same case. So test was slightly redundant.